source/Engine.cpp

```diff
@@ -2035,7 +2034,7 @@ void Engine::HandleMouseClicks()
 	double clickRange = 50.;
 	shared_ptr<Ship> clickTarget;
 	for(shared_ptr<Ship> &ship : ships)
-		if(ship->GetSystem() == playerSystem && &*ship != flagship && ship->IsTargetable())
+		if(ship->GetSystem() == playerSystem && &*ship != flagship && (ship->IsTargetable() || ship->IsYours()))
```

Comment:
Why let the player click on their ships that aren't targettable anymore.

Reply:
caus its yours but alr ig it's more of a bugfix that should be split from this

Reply:
idk if you're ship is say exploding why should it be possible to target it?